### PR TITLE
Refresh _ru\tour\named-argument.md

### PR DESCRIPTION
**Pull request by artemkorsakov**
Refresh `_ru\tour\named-argument.md` page

### CODE REVIEW

**Comment by artemkorsakov** [comment]
@kotobotov @DespiteDeath please, review this PR

File: _ru/tour/named-arguments.md [hunk 1]
```diff
@@ -11,19 +11,35 @@ prerequisite-knowledge: function-syntax
 
 При вызове методов можно конкретно указывать название задаваемого аргумента следующим образом:
 
+{% tabs named-arguments-when-good %}
+
+{% tab 'Scala 2 и 3' for=named-arguments-when-good %}
 ```scala mdoc
-def printName(first: String, last: String): Unit = {
+def printName(first: String, last: String): Unit =
```
Comment (DespiteDeath) [inline _ru/tour/named-arguments.md]: Maybe add newline before and after ``` due to lint rules for `.md` files here and below (and in [2782](https://github.com/scala/docs.scala-lang/pull/2782), [2783](https://github.com/scala/docs.scala-lang/pull/2783))

**Review by DespiteDeath — APPROVED** [review]

**Comment by artemkorsakov** [comment]
@julienrf could you please merge this approved PR?

**Comment by julienrf** [comment]
Thank you!

**Pull request merged**